### ISSUE TITLE
[google_maps_flutter]': GoogleMapsBase' not found 

### ISSUE THREAD

**Issue by ndjido** [flutter/flutter#26144]
Hi there,
In ios, I'm getting the following fatal error:
```
fatal error: module
    'GoogleMapsBase' not found
```
In my `pubspec.yaml` file, I added the plugin as so `google_maps_flutter: ^0.0.3+3`.
My `AppDelegate.swift` contains the following:

```swift
import UIKit
import Flutter
import GoogleMaps

@UIApplicationMain
@objc class AppDelegate: FlutterAppDelegate {
  override func application(
    _ application: UIApplication,
    didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?
  ) -> Bool {
    GeneratedPluginRegistrant.register(with: self)
    GMSServices.provideAPIKey("MY_KEY")
    return super.application(application, didFinishLaunchingWithOptions: launchOptions)
  }
}
```

Is there anyone witnessing the same issue?

**Comment by zoechi** [comment]
Please consider asking support questions in one of the other channels listed at http://flutter.io/support .

Please add the output of `flutter doctor -v`.

**Comment by akipmaulana** [comment]
Hello @ndjido, I already solved this issue by updating cocoa pod version, 1.4.0 to 1.5.3. 
hopefully can help you.

**Comment by zoechi** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!
Thanks for your contribution.

**Comment by rosulg** [comment]
Had the same issue with CocoaPods version 1.5.0. Resolved it by updating to 1.6.1.

It should be mentioned that I got it running with version 1.5.0 also, but in that case I manually set the 'Target Membership' of 'GoogleMapsBase.framework' to 'google_maps_flutter'.
